tests/test_core.py

```diff
@@ -4798,7 +4798,7 @@ def test_fs_trackingdelegate(self):
   @also_with_noderawfs
   def test_fs_writeFile(self, js_engines=None):
     self.emcc_args += ['-s', 'DISABLE_EXCEPTION_CATCHING=1'] # see issue 2334
-    src = path_from_root('tests', 'fs', 'test_writeFile.cc')
+    src = path_from_root('tests', 'fs', 'test_writeFile.cpp')
```

Comment:
Shouldn't .cc be picked up as a C++ extension anyway? Or is this just for consistency in the testsuite

Reply:
Just for consistency.  We use .cpp for C++ source files in emscripten it seems.  This was the odd one out.